docs/resources/container.md

```diff
@@ -0,0 +1,134 @@
+
+- `max_scale` - (Optional) The maximum of number of instances this container can scale to. Default to 20.
+
+- `memory_limit` - (Optional) The memory computing resources in MB to allocate to each container. Defaults to 128.
+
+- `cpu_limit` - (Optional) The amount of vCPU computing resources to allocate to each container. Defaults to 70.
+
+- `timeout` - (Optional) The maximum amount of time in seconds during which your container can process a request before we stop it. Defaults to 300s.
+
+- `privacy` - (Optional) The privacy type define the way to authenticate to your container. Please check our dedicated [section](https://developers.scaleway.com/en/products/containers/api/#protocol-9dd4c8).
+
+- `registry_image` - (Optional) The registry image address. e.g: **"rg.fr-par.scw.cloud/$NAMESPACE/$IMAGE"**.
+
+- `max_concurrency` - (Optional) The maximum number of simultaneous requests your container can handle at the same time. Defaults to 50.
+
+- `domain_name` - (Optional) The container domain name.
+
+- `protocol` - (Optional) The communication [protocol](https://developers.scaleway.com/en/products/containers/api/#protocol-9dd4c8) http1 or h2c. Defaults to http1.
+
+- `port` - (Optional) The port to expose the container. Defaults to 8080.
+
+- `deploy` - (Optional) Boolean controlling whether the container is on a production environment.
+
+Note that if you want to use your own configuration, you must consult our configuration [restrictions](https://www.scaleway.com/en/docs/compute/containers/reference-content/containers-limitations/#configuration-restrictions) section.
+
+## Attributes Reference
+
+In addition to all above arguments, the following attributes are exported:
+
+- `region` - (Defaults to [provider](../index.md#region) `region`) The [region](../guides/regions_and_zones.md#regions) in which the container was created.
+- `status` - The container status.
+- `cron_status` - The cron status of the container.
+- `error_message` - The error message of the container.
+
+## Import
+
+Container can be imported using the container_name, e.g.,
+
+```bash
+$ terraform import scaleway_container.main fr-par/11111111-1111-1111-1111-111111111111
+```
+
+## Protocols
+
+The supported protocols are:
+
+* `h2c`: HTTP/2 over TCP.
+* `http1`: Hypertext Transfer Protocol.
+
+**Important:** For details about the protocols check [this](https://httpd.apache.org/docs/2.4/howto/http2.html)
+
+## Privacy
+
+By default, creating a container will make it `public`, meaning that anybody knowing the endpoint could execute it.
+A container can be made `private` with the privacy parameter.
+
+Please check our [authentication](https://developers.scaleway.com/en/products/containers/api/#protocol-9dd4c8) section
+
+## Memory and vCPUs configuration
+
+The vCPU represents a portion or share of the underlying, physical CPU that is assigned to a particular virtual machine (VM).
```

Comment:
How is this portion measured? What is the maximum vCPU possible in theory? Is it a gauge?